pkg/pool-manager/pool_test.go

```diff
@@ -106,6 +107,30 @@ var _ = Describe("Pool", func() {
 			table.Entry("Invalid address: 01:FF:00:00:00:00, the first octet is not 02, 06, 0A or 0E", "01:FF:00:00:00:00", true),
 			table.Entry("Invalid address: FF:FF:00:00:00:00, the first octet is not 02, 06, 0A or 0E", "FF:FF:00:00:00:00", true),
 		)
+
+		table.DescribeTable("should check that a mac pool size is reported correctly", func(startMacAddr, endMacAddr string, expectedSize float64, needToSucceed bool) {
+			startMacAddrHW, err := net.ParseMAC(startMacAddr)
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
Add "should..."

Reply:
Isn't this already check at `checkRange` ?

Reply:
nope. `checkRange` only checks if one is not bigger than the other without parsing to int.